routers/repo/webhook.go

```diff
@@ -150,7 +165,7 @@ func ParseHookEvent(form auth.WebhookForm) *models.HookEvent {
 	}
 }
 
-// WebHooksNewPost response for creating webhook
+// WebHooksNewPost response for creating Gitea webhook
```

Comment:
Only gitea webhook?

Reply:
Yep, in `routes.go` only the new Gitea hook uses this handler:
```
			m.Get("/:type/new", repo.WebhooksNew)
			m.Post("/gitea/new", bindIgnErr(auth.NewWebhookForm{}), repo.WebHooksNewPost)
			m.Post("/gogs/new", bindIgnErr(auth.NewGogshookForm{}), repo.GogsHooksNewPost)
			m.Post("/slack/new", bindIgnErr(auth.NewSlackHookForm{}), repo.SlackHooksNewPost)
```

I suppose the method should be renamed really.

Reply:
Can you rename it?

Reply:
Sure :+1: